README.md

```diff
@@ -1,2 +1,68 @@
 # app.js
-GitHub Apps toolset for Node.js
+
+> Official GitHub Platform for GitHub Apps in Node.js
+
+## GitHub Apps
+
+`@octokit/app.js` allows to authenticate as GitHub Apps, installations, listen
+to webhooks, and respond to them using the GitHub API.
+
+### Authenticating as an App
+
+In order to authenticate as a GitHub App, you need to generate a Private Key
+and use it to sign a JSON Web Token (jwt) and encode it. 
+
+```js
+const App = require('@octokit/app.js')
+
+const app = new App({id, privateKey})
+const appAuth = await app.appAuth()
+```
+
+### Authenticating as an Installation
+
+Once you have authenticated as a GitHub App, `auth` here, you can use that
+in order to request an installation access token. This token is scoped for
+your specific app and expires after an hour.
+
+```js
+const app = require('@octokit/app.js')
+
+const app = new App({id, privateKey})
+const appAuth = await app.appAuth()
+const installation = await app.requestToken(appAuth, installationId)
```

Comment:
I don’t think that the `app.appAuth()` call is necessary, I think it can be done internally within `app.requestToken()`

I’d also suggest to pass `installationId` as an object (`{installationId: 123}`), it makes the API more flexible if we want to pass more than one option in future